benchmarks/patches/mismatched_batch_size.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/torchbenchmark/util/model.py b/torchbenchmark/util/model.py
+index 8593ba4c..57fef507 100644
+--- a/torchbenchmark/util/model.py
++++ b/torchbenchmark/util/model.py
+@@ -182,6 +182,7 @@ class BenchmarkModel(metaclass=PostInitProcessor):
+
+         # use the device suggestion on CUDA inference tests, key should be either eval_batch_size or train_batch_size
+         device_batch_size_key = f"{self.test}_batch_size"
++        current_device_name = 'NVIDIA A100-SXM4-40GB'
```

Comment:
Maybe an in-patch comment on why this is needed?

Reply:
Done, I additionally added the info how to apply the patch into the README file (cc: @vanbasten23)